server/embed/config.go

```diff
@@ -449,11 +452,11 @@ type configYAML struct {
 
 // configJSON has file options that are translated into Config options
 type configJSON struct {
-	ListenPeerUrls       string `json:"listen-peer-urls"`
-	ListenClientUrls     string `json:"listen-client-urls"`
-	ListenClientHttpUrls string `json:"listen-client-http-urls"`
-	AdvertisePeerUrls    string `json:"initial-advertise-peer-urls"`
-	AdvertiseClientUrls  string `json:"advertise-client-urls"`
+	ListenPeerURLs       string `json:"listen-peer-urls"`
+	ListenClientURLs     string `json:"listen-client-urls"`
+	ListenClientHTTPURLs string `json:"listen-client-http-urls"`
+	AdvertisePeerURLs    string `json:"initial-advertise-peer-urls"`
+	AdvertiseClientURLs  string `json:"advertise-client-urls"`
```

Comment:
As stated in the PR description, these are exported fields from an unexported struct. It's internally used to marshal/unmarshal JSONs. So, it should be fine to rename.

Reply:
Right